sql-and-jdbc/src/main/java/org/example/PersonRepository.java

```diff
@@ -0,0 +1,16 @@
+package org.example;
+
+import java.sql.SQLException;
+import java.util.List;
+
+public interface PersonRepository {
+    void addPerson(Person person) throws SQLException;
+
+    void getPersonById(int numberInput) throws SQLException;
```

Comment:
if you say getPerson, I need a method that actually retrieves that person :) i don't know how i should work with this method otherwise

Reply:
also, pls use Optional to avoid NPEs